.github/workflows/build.yml

```diff
@@ -25,7 +25,7 @@ jobs:
 
     - uses: actions/setup-java@v2
       with:
-        distribution: 'adopt'
+        distribution: 'zulu'
```

Comment:
```suggestion
        distribution: 'temurin'
```
See comment. We prefer to use Eclipse Temurin

Reply:
I've updated both workflows to use Temurin as the distribution for setup-java GH actions. 👍🏼